### PR TITLE
Update shortbow to use require and add global require function

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 85c9551e8bfe0497f7996c59e591aaf8307157a8 available at: https://gist.github.com/0a70262b66a90fddd319458c4cb2cbe4<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9974-85c9551.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9974-85c9551.dmg

**Comment by huffman** [comment]
To test:

 1. Run https://raw.githubusercontent.com/huffman/hifi/85c9551e8bfe0497f7996c59e591aaf8307157a8/unpublishedScripts/marketplace/shortbow/spawnShortbow.js
 1. Play shortbow
 1. Ensure that everything is working

File: unpublishedScripts/marketplace/shortbow/enemyClientEntity.js [hunk 1]
```diff
@@ -6,10 +6,8 @@
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 //
 
-/* globals utils */
-
 (function() {
-    Script.include('utils.js');
+    const utils = require('./utils.js');
```
Comment (thoys) [inline unpublishedScripts/marketplace/shortbow/enemyClientEntity.js]: use var

File: unpublishedScripts/marketplace/shortbow/shortbowGameManager.js [hunk 2]
```diff
@@ -6,9 +6,7 @@
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 //
 
-/* globals ShortbowGameManager:true, utils */
-
-Script.include('utils.js');
+const utils = require('./utils.js');
```
Comment (thoys) [inline unpublishedScripts/marketplace/shortbow/shortbowGameManager.js]: use var

File: unpublishedScripts/marketplace/shortbow/spawnShortbow.js [hunk 3]
```diff
@@ -7,10 +7,10 @@
 //
 /* globals utils, SHORTBOW_ENTITIES, TEMPLATES:true */
 
-Script.include('utils.js');
-Script.include('shortbow.js');
-Script.include('shortbowGameManager.js');
-TEMPLATES = SHORTBOW_ENTITIES.Entities;
+const utils = require('./utils.js');
```
Comment (thoys) [inline unpublishedScripts/marketplace/shortbow/spawnShortbow.js]: use var

File: unpublishedScripts/marketplace/shortbow/spawnShortbow.js [hunk 4]
```diff
@@ -7,10 +7,10 @@
...
+const shortbowEntities = require('./shortbow.js');
```
Comment (thoys) [inline unpublishedScripts/marketplace/shortbow/spawnShortbow.js]: use var

File: unpublishedScripts/marketplace/shortbow/enemyServerEntity.js [hunk 5]
```diff
@@ -6,10 +6,8 @@
 //  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 //
 
-/* globals utils */
-
 (function() {
-    Script.include('utils.js');
+    const utils = require('./utils.js');
```
Comment (thoys) [inline unpublishedScripts/marketplace/shortbow/enemyServerEntity.js]: use var

**Comment by Nex-Pro** [comment]
Stringing my bow for this one...
Testing

**Comment by Nex-Pro** [comment]
This build is a FAIL on my system.

 @huffman 


I'm getting some really strange rendering issue on this build. I've tried restarting etc but it won't go away.
I initially installed on drive E:  Problem shows up in desktop and HMD
re-downloaded.
reinstalled after removing all folders. Problem persists.
reinstalled on drive C: problem persists in desktop and HMD see screenshots.

I am not having this issue in dev build or other builds and have never seen this issue previously.

hifi-snap-by--on-2017-05-03_20-29-13
hifi-snap-by--on-2017-05-03_20-23-22
hifi-snap-by--on-2017-05-03_20-23-31
hifi-snap-by--on-2017-05-03_20-23-34
hifi-snap-by--on-2017-05-03_20-23-40

**Comment by hifi-gustavo** [comment]
Build console output for commit: 507036aa527632b86c63e41344a353a6f9ce5848 available at: https://gist.github.com/0a70262b66a90fddd319458c4cb2cbe4<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9974-507036a.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9974-507036a.dmg

**Comment by huffman** [comment]
@thoys I addressed your comments
@Nex-Pro This is ready to test again

**Comment by thoys** [comment]
Looks good @huffman

**Comment by huffman** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: acd34999d74916e9f3665649567aae10024038c0 available at: https://gist.github.com/0a70262b66a90fddd319458c4cb2cbe4<br/><br/>Installers available for download at: <br/>Windows: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9974-acd3499.exe<br/>OS X: http://cachefly.highfidelity.com/HighFidelity-Beta-PR9974-acd3499.dmg

**Comment by Nex-Pro** [comment]
Testing this Win10 and Vive

**Comment by Nex-Pro** [comment]
### Fail.

game will not start.

Tried several times and restarted domain.

Also collisions are broken on the walkway.

Initially had no sound.

got sound working by using "Echo Server Audio"

Then  background sounds started working.  Strange.

Let me know if you want any logs.

**Comment by ZappoMan** [comment]
This PR is QA Failing for over 3 months. Closing until developer has fixed.

**Pull request closed without merging**